services/movie_session.py

```diff
@@ -0,0 +1,61 @@
+from typing import Optional
+
+from db.models import MovieSession
+from django.db.models import QuerySet
+
+import datetime
+
+
+def create_movie_session(
+        movie_show_time: datetime,
```

Comment:
Here is also datetime.datetime
Please do self-review.....
You do:
import datetime
that means datetime is module